src/rules/sw.toml

```diff
@@ -0,0 +1,33 @@
+min_trimmed_length = 3
+min_word_count = 3
+max_word_count = 14
+min_characters = 0
+may_end_with_colon = false
+quote_start_with_letter = true
+needs_punctuation_end = false
+needs_letter_start = true
+needs_uppercase_start = false 
+disallowed_symbols = [
```

Comment:
Might be worth to look into `allowed_symbols_regex` to turn this around. https://github.com/common-voice/cv-sentence-extractor#using-language-rules

Reply:
Hi, Michael;
We ended up erring on the conservative side; there's a fair number of English words and place-names with pretty high frequency. Clicking through the 'random pages' feature also indicates that a decent proportion of the articles are stubs about places; a few sentences with low-frequency place-names, which will tend to be filtered.

https://sw.wikipedia.org/wiki/Maalum:UkurasawaBahati

Reply:
Thanks for the answer. So you feel confident that 9189 is the maximum while still keeping quality to a good level? If so, I'm fine with that, but I want to make sure we make the best of it, as we can't re-run the extract on these articles more than once.

Reply:
Yeah, we ran the extractor locally at (iirc) frequencies 60, 70, 80, 90, and 100, and then examined the sentences in the diffs.

Suppose we have D sentences in the diff between frequencies F and F', and N_F total extracted sentences with frequency F.  We examine the sentences in the diff, and estimate that they're bad with probability p. Then p * D / N_F is a lower bound on the ratio of bad sentences in the full set of sentences at frequency F, so we want p * D / N_F < 0.05. We settled on frequency 80 as the sweet spot for getting past lots of English words and place names which were dragging down quality. I think Kevin has the spreadsheets where we did the analysis; my memory is that we pick up a couple thousand sentences if we dial back to frequency 70, but would need to develop further word block-lists to bring the miss-rate under control.

Reply:
Wow, nice data based approach!